### ISSUE TITLE
Question - running on the .net core

### ISSUE THREAD

**Issue by joeblew99** [MicrosoftResearch/Dryad#7]
Now that .net core runs on Linux and plinq is included, that means that dryad could be effectively used on a massive compute cluster.

Seems like a very nice opportunity. Can anyone of the tram comment on this ?

https://github.com/dotnet/corefx

**Comment by RahulNET** [comment]
I agree with you joeblew99.  I think it could be a very interesting opportunity to bring .NET into the game of Hadoop ecosystem.  To be able to write **YARN aware programs in PLINQ** could be really powerful similar to [Cascading](http://www.cascading.org/) or [Cascalog](http://cascalog.org/).  I would be very keen to contribute on this.  Let me know.

**Comment by frankmcsherry** [comment]
Microsoft Research closed the lab that produced this work last September. The team members are now mostly at Google, and (I think) probably not in a position to develop or support Microsoft projects.

When we left, both Dryad and Naiad ran just fine on Yarn clusters and worked just fine with HDFS. If you have more questions, I recommend getting in touch with someone at Microsoft. I suspect Mark Staveley could give you a definitive thumbs up / thumbs down signal.

But, good to hear that you like what the lab did.

**Comment by joeblew99** [comment]
Well since then things have moved on. I am developing 100% of the stack in
golang.

Its going very well. Go can run on android and ios now. On the server side
its very easy.
.the garbage collector is my main hurdle

J

**Comment by michaelisard** [comment]
In case anyone else is interested in returning to the codebase: as Frank says, things run pretty smoothly today on YARN, but only on Windows. The problem running on Linux is that Dryad and the YARN interop layer both include managed C++. I have not been following closely, but it used to be difficult to get managed C++ to work on Linux without some porting effort, and nobody to my knowledge has tried to do that.

I don’t think Mark Staveley is in a role to support this stuff any more, and I don’t know of anyone else at Microsoft who has picked it up. Someone could always fork the code…
